packages/ovmf/build.sh

```diff
@@ -43,6 +45,7 @@ termux_step_make_install() {
 }
 
 termux_step_install_license() {
-	install -Dm600 $TERMUX_PKG_BUILDER_DIR/License.txt \
-		$TERMUX_PREFIX/share/doc/$TERMUX_PKG_NAME/LICENSE.txt
+	mkdir -p $TERMUX_PREFIX/share/doc/$TERMUX_PKG_NAME
+	mv $TERMUX_PREFIX/share/licenses/edk2-ovmf/* $TERMUX_PREFIX/share/doc/$TERMUX_PKG_NAME/
+	rm -rf $TERMUX_PREFIX/share/{licenses,LICENSES}
```

Comment:
Kind of wild... Should be in post-massage if ever needed.

Reply:
`/usr/share/licenses` is the folder where fedora packages use to contain licenses, but for Termux, the licenses are saved into `$PREFIX/share/doc`, so I think these folders should be deleted. I just confirm that there is no need to deal with `$PREFIX/share/LICENSES` folder because it is empty.

Reply:
That is a dangerous assumption. AFAIK the directory is not reserved by our system and users can use it as they wish. Please do not delete the entire directory, ever, please. I will refactor it on my own if it is not improved.